app/assets/javascripts/components/activity/did_you_know_handler.cjsx

```diff
@@ -39,6 +37,7 @@ DidYouKnowHandler = React.createClass(
         Show My Courses Only
       </label>
       <ActivityTable
+        loading={@state.loading}
```

Comment:
I assume this adds that loading animation while it's waiting for data from the server. We should do the same thing for the plagiarism and recent edits handlers.